internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```
@@ -48,6 +48,11 @@
     StartPeriod: {{.HealthCheck.StartPeriod}}
     Timeout: {{.HealthCheck.Timeout}}
 {{- end}}
+{{- if .Storage}}
+{{- if .Storage.ReadonlyRootFS}}
```

nit: you can just club 2 conditions together.
```suggestion
{{- if and .Storage .Storage.ReadonlyRootFS}}
```